spec.html

```diff
@@ -5893,15 +5903,16 @@
 
     <!-- es6num="8.2.3" -->
     <emu-clause id="sec-setrealmglobalobject" aoid="SetRealmGlobalObject">
-      <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_ )</h1>
-      <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_ and _globalObj_ performs the following steps:</p>
+      <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_, _thisValue_ )</h1>
+      <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_, _globalObj_, and _thisValue_ performs the following steps:</p>
```

Comment:
It's not actually optional; all places in the spec call it with all three arguments. Sometimes *undefined* is used as a default-value signifier.